docs/languages/python.md

```diff
@@ -0,0 +1,8 @@
+# Python
```

Comment:
@awilkes Python is now supported by two languages, so I created a language page.

Reply:
Awesome!